designer/report-designer/src/main/java/org/pentaho/reporting/designer/core/actions/elements/MorphAction.java

```diff
@@ -106,6 +112,10 @@ public void actionPerformed( final ActionEvent e ) {
       if ( ObjectUtilities.equal( visualElement.getElementTypeName(), targetElementType.getMetaData().getName() ) ) {
         continue;
       }
+      if ( COMPLEX_ELEMENT_SET.contains( targetElementType.getMetaData().getName() )
```

Comment:
Morphing has two inputs: What element do I start with, and what element type do I end up with. 

The menu choice gives you the target (ie: "Morph the selected element into a .. (label, text-field, etc)"). But the source of your morph is the current selection, and that is changing all the time. 

For target types that are eternally disabled, like the complex elements like subreports and groups, we do not generate the menu in the first place. We still have to handle the case where a user selects one of those elements as its source element and expects that to convert to something else.